addon/mixins/frost-list-sorting-mixin.js

```diff
@@ -3,35 +3,25 @@ const {
   Mixin,
   on
 } = Ember
-import computed from 'ember-computed-decorators'
+import {normalizeSort, listDefaultSort} from 'ember-frost-list/utils/utils'
 import FrostListCoreMixin from 'ember-frost-list/mixins/frost-list-core-mixin'
 
 export default Mixin.create(FrostListCoreMixin, {
   // == Event =================================================================
   initListSortingMixin: on('init', function () {
-    this.set('queryParams', ['listConfig.sorting.active'])
     Ember.defineProperty(this, 'sortableProperties', Ember.computed.alias('listConfig.sorting.properties'))
     Ember.defineProperty(this, 'activeSorting', Ember.computed.alias('listConfig.sorting.active'))
   }),
 
-  // == Computed Properties ====================================================
-  sortedItems: Ember.computed.sort('statefulListItems', 'activeSortingString'),
-
-  @computed('activeSorting')
-  activeSortingString (activeSorting) {
-    if (!activeSorting) return []
-    return activeSorting.map((sortProperty) => {
-      return `record.${sortProperty.value}${sortProperty.direction}`
-    })
-  },
-
   // == Actions ================================================================
   actions: {
-    sortItems (sortItems) {
-      let activeSorting = sortItems.map(function (item) {
+    sortItems (sortProperties) {
+      let filteredSortProperties = sortProperties.map(function (item) {
         return {value: item.value, direction: item.direction}
       })
-      this.set('activeSorting', activeSorting)
+      let normalizedSortProperties = normalizeSort(filteredSortProperties)
+      const dataKey = this.get('listConfig.items')
+      this.set(dataKey, listDefaultSort(this.get(dataKey), normalizedSortProperties))
```

Comment:
Shouldn't this be conditional on someone providing their own sort function?  I.E. can you override the default or not sort on the client at all?